Podcast/PlayerViewController.swift

```diff
@@ -12,9 +13,33 @@ class PlayerViewController: TabBarAccessoryViewController, PlayerDelegate, Playe
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        
         view.backgroundColor = .clear
         
+        backgroundImageView = ImageView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height))
+        backgroundImageView.contentMode = .scaleAspectFill
+        view.addSubview(backgroundImageView)
+        backgroundImageView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+        
+        let blurEffect = UIBlurEffect(style: UIBlurEffectStyle.extraLight)
+        let blurEffectView = UIVisualEffectView(effect: blurEffect)
+        blurEffectView.frame = view.frame
+        blurEffectView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        blurEffectView.alpha = 0.5
+        view.addSubview(blurEffectView)
+        
+        let gradientView = UIView()
```

Comment:
seems like the second time we are doing this gradient view... let's abstract this away then
create a class GradientLayer (or equivalent name)